src/Illuminate/Database/Query/Builder.php

```diff
@@ -1318,27 +1318,29 @@ public function find($id, $columns = array('*'))
 	}
 
 	/**
-	 * Pluck a single column's value from the first result of a query.
+	 * Get a single column's value from the first result of a query.
 	 *
 	 * @param  string  $column
 	 * @return mixed
 	 */
 	public function value($column)
 	{
-		return $this->pluck($column);
+		$result = (array) $this->first(array($column));
+
+		return count($result) > 0 ? reset($result) : null;
 	}
 
 	/**
-	 * Pluck a single column's value from the first result of a query.
+	 * Alias for the "value" method.
```

Comment:
https://github.com/laravel/framework/blob/5.0/src/Illuminate/Database/Query/Builder.php#L1187
https://github.com/laravel/framework/blob/5.0/src/Illuminate/Database/Query/Builder.php#L1187

You could argue that those are not the same, since the alias description already describes what the method does. But still...